OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -2,10 +2,7 @@
   <PropertyGroup>
     <RootNamespace>OpenRA</RootNamespace>
   </PropertyGroup>
-  <PropertyGroup Condition="'$(Mono)' != ''">
-    <DefineConstants>MONO</DefineConstants>
-  </PropertyGroup>
-  <ItemGroup Condition="'$(Mono)' == ''">
+  <ItemGroup Condition="'$(TargetFramework)' != 'netstandard2.1'">
```

Comment:
Thinking about it, maybe we should keep this? Otherwise this needs to change every time we change https://github.com/OpenRA/OpenRA/blob/b67954451aa0401c04d01916ce6d044c272c96a3/Directory.Build.props#L21 doesn't it?